src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -52,17 +54,32 @@ export class HuiCardOptions extends LitElement {
           color: var(--primary-color);
           font-weight: 500;
         }
+        paper-button.warning:not([disabled]) {
+          color: var(--google-red-500);
+        }
       </style>
       <slot></slot>
-      <div><paper-button @click="${this._editCard}">EDIT</paper-button></div>
+      <div>
+        <paper-button class="warning" @click="${this._deleteCard}"
+          >DELETE</paper-button
+        ><paper-button @click="${this._editCard}">EDIT</paper-button>
+      </div>
     `;
   }
-  private _editCard() {
+  private _editCard(): void {
     showEditCardDialog(this, {
       cardConfig: this.cardConfig!,
       reloadLovelace: () => fireEvent(this, "config-refresh"),
     });
   }
+  private _deleteCard(): void {
+    confDeleteCard(
+      this.hass!,
+      this.cardConfig!,
+      () => fireEvent(this, "config-refresh"),
+      () => this._editCard()
```

Comment:
This makes no sense, can better check for an ID before calling delete

Reply:
Yep, disable button when no ID and wrap it in a tooltip saying "Card has no ID"

Reply:
Or show the migrate dialog, like this._editCard() does?